spec/unit/provider/package/windows/exe_package_spec.rb

```diff
@@ -34,7 +34,6 @@
     {
       'DisplayName'      => ['My App', ''],
       'UninstallString'  => ['E:\uninstall.exe', ''],
-      'SystemComponent'  => [nil, 1],
       'WindowsInstaller' => [nil, 1],
       'ParentKeyName'    => [nil, 'Uber Product'],
       'Security Update'  => [nil, 'KB890830'],
```

Comment:
I'm actually on the fence about this.. I can see reasons for wanting puppet to report on all installed packages, e.g. `puppet resource package` even if it's not something that puppet can uninstall.